example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -82,7 +81,11 @@ auto create_args(int argc, char* argv[])
         .insert("prec", "fp16", "data type. fp16/bf16/fp8/bf8")
         .insert("warmup", "50", "number of iterations before benchmark the kernel")
         .insert("repeat", "100", "number of iterations to benchmark the kernel")
-        .insert("timer", "gpu", "gpu:gpu timer, cpu:cpu timer");
+        .insert("timer", "gpu", "gpu:gpu timer, cpu:cpu timer")
+        .insert(
+            "split_k",
+            "1",
+            "splitK value, set to 0 to run the set of the values {1, 2, 4, 8, 16, 32, 64, 128}");
```

Comment:
I don't see testing over this set of values and I don't think this is needed in example.